### PR TITLE
NXP-27186: Fix editing dates in webui-ftests framework - 10.10

### PR DESCRIPTION
**Pull request by FabioFSantos**

### CODE REVIEW

File: nuxeo-web-ui-ftest/test/features/step_definitions/support/fixtures/layouts.js [hunk 1]
```diff
@@ -70,7 +70,11 @@ global.fieldRegistry.register('nuxeo-date-picker',
   element => moment(element.element('vaadin-text-field').getValue(), global.dateFormat)
     .format(global.dateFormat),
   (element, value) => {
-    element.element('vaadin-text-field').click();
+    const date = element.element('vaadin-text-field');
+    if (date.getValue() !== null && date.getValue() !== '') {
```
Comment (Gabez0r) [inline nuxeo-web-ui-ftest/test/features/step_definitions/support/fixtures/layouts.js]:
```suggestion
    if (date.getValue()) {
```

**Review by Gabez0r — APPROVED** [review]

**Comment by Gabez0r** [comment]
👍

File: nuxeo-web-ui-ftest/test/features/step_definitions/support/fixtures/layouts.js (hunk 1 above)
Comment (nmpcunha) [reply nuxeo-web-ui-ftest/test/features/step_definitions/support/fixtures/layouts.js]: Agree!

**Comment by nmpcunha** [comment]
👍

**Review by nmpcunha — APPROVED** [review]

**Pull request merged**